src/openforms/formio/components/custom.py

```diff
@@ -12,6 +12,7 @@
 
 from openforms.authentication.constants import AuthAttribute
 from openforms.config.models import GlobalConfiguration
+from openforms.contrib.brk.validators import AddressValueSerializer
```

Comment:
I think it should be the other way around - BRK validators should use the serializer defined here (so move the serializer into this module please)

The implementation is also missing the `plugins` validation being applied in the backend validation.